front/lib/resources/group_resource.ts

```diff
@@ -125,7 +125,7 @@ export class GroupResource extends BaseResource<GroupModel> {
       where: {
         workspaceId: key.workspaceId,
         [Op.or]: [
-          { kind: key.isSystem ? "system" : "global" },
+          { kind: key.isSystem ? ["system", "global"] : "global" },
```

Comment:
Is this correct @spolu @tdraier  ? System key should have access to both system and global groups.

Reply:
That looks not crazy.

cc @flvndvd given the TODO(GROUPS_INFRA)